Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -148,7 +148,7 @@ public IEnumerator WaitAndStartNextRound()
         yield return new WaitForSeconds(roundEndDelay);
         // This needs to be called after inputs are set at start the first time this is needed.
         PlayerInputManagerController.Singleton.ChangeInputMaps("FPS");
-        SceneManager.LoadScene("DemoArena");
+        SceneManager.LoadScene("Cratertown");
```

Comment:
Maybe we should rename this to PascalCase?

Reply:
Yeah I thought of it but forgor to do it :skull: